packages/token-sdk/package.json

```diff
@@ -1,21 +1,22 @@
 {
   "name": "@orca-so/token-sdk",
-  "version": "0.1.0",
+  "version": "0.1.5",
```

Comment:
Bumping to `0.1.5` version. I published a few versions for quicker testing with Github Actions. I'm likely going to need to publish more versions for testing. I could either publish with an alpha tag - e.g. `0.1.5.alpha.0`, or we can bump the minor version to `0.2.0` when things are more stable.

Reply:
https://www.npmjs.com/package/@orca-so/token-sdk

Reply:
@terranmoccasin 

+1👍: 0.1.5-alpha.N

IMO, even users who prefer to use the latest version do not want to use a version for testing purposes.

https://github.com/orca-so/whirlpools/releases/tag/0.6.0
https://semver.org/